_conferences/fosdem-2020.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "FOSDEM - Kotlin DevRoom"
```

Comment:
```suggestion
name: "FOSDEM"
```

I wouldn't pigeon hole attendees to this room since there are potentially other relevant tracks like game development.

Reply:
True, but the CFP link I added is the one from the Kotlin DevRoom. Maybe I should create two conference entry 🤔? One for generic FOSDEM and another for the Kotlin DevRoom?

Reply:
What about this link? https://fosdem.org/2020/news/2019-10-01-accepted-developer-rooms/

Reply:
What about this link? https://fosdem.org/2020/news/2019-10-01-accepted-developer-rooms/

Reply:
Yup let's go for that